pkg/gardenlet/controller/seed/seed/components.go

```diff
@@ -97,14 +97,26 @@ func defaultIstio(
 		return nil, err
 	}
 
+	gardenSeed := seed.GetInfo()
```

Comment:
I think it makes sense to add a document (and maybe even a nice Excalidraw diagram) about this architecture like in the VPN-HA PR :)

Reply:
Updated istio documentation and also added a diagram. Will be included in the next commit.